internal/network/manifests_generator.go

```diff
@@ -24,15 +25,22 @@ type ManifestsGeneratorAPI interface {
 	AddChronyManifest(ctx context.Context, log logrus.FieldLogger, c *common.Cluster) error
 	AddDnsmasqForSingleNode(ctx context.Context, log logrus.FieldLogger, c *common.Cluster) error
 	AddDisableVmwareTunnelOffloading(ctx context.Context, log logrus.FieldLogger, c *common.Cluster) error
+	AddTelemeterManifest(ctx context.Context, log logrus.FieldLogger, c *common.Cluster) error
+}
+
+type Config struct {
+	ServiceBaseURL string `envconfig:"SERVICE_BASE_URL"`
```

Comment:
Why do you need this config?
Why not just pass the relevant value (staging/prod) to `AddTelemeterManifest`?

Reply:
Not sure I get what you mean, the caller doesn't now this value, it should be passed through the whole stack from bminventory pkg.

See https://coreos.slack.com/archives/CUPJTHQ5P/p1621515876386800